tests/client/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -xe
+
+if [ $# -eq 1 ]; then
```

Comment:
Does the go syntax support running multiple tests ? E.g. `./sdb_test -test.run test1 test2` ?

Reply:
I did not test yet. But it supports regex.

Reply:
When I tested, It does not support multiple arguments.
But If you want to test multiple, then, using regexp will helpful
```
  -test.run regexp
    	run only tests and examples matching regexp
```